src/app/clients/add/formFunctions.ts

```diff
@@ -0,0 +1,281 @@
+import React from "react";
+import { SelectChangeEvent } from "@mui/material";
+import { booleanGroup } from "@/components/DataInput/inputHandlerFactories";
+import supabase, { InsertSchema } from "@/supabase";
+import { Database } from "@/database_types_file";
+
+type Field = Fields[keyof Fields];
+type OnChange = (event: React.ChangeEvent<HTMLInputElement> | SelectChangeEvent) => void;
+type OnChangeCheckbox = (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
are these the same as `changeEventHandler` in `inputHandlerFactories`?

Reply:
yep. I've changed it so that it uses the types exported from `inputHandlerFactories`